packages/portfolio/src/adapters/dullahan-api/types.ts

```diff
@@ -35,6 +35,11 @@ export type DullahanApiTokenActivityUpdates = Readonly<{
   price24h: DullahanApiTokenActivityRecord
 }>
 
+export type DullahanApiTokenActivityUpdatesStatusCode = number
```

Comment:
build the records `[HttpStatus.OK, Portfolio.TokenActivity] | [404 | 500, string]`